src/subscription_manager/cli_command/abstract_syspurpose.py

```diff
@@ -262,7 +262,7 @@ def _are_provided_values_valid(self, values):
                     print(
                         _(
                             "Warning: This organization does not have any subscriptions that provide a "
-                            'system purpose "{attr}".  This setting will not influence auto-attaching '
```

Comment:
syspurpose was actually a key component in the automatic attachment, not in the manual one; this string change is not correct, as "auto-attaching" here is the proper action

if anything, the whole message should be reviewed in light of a SCA-only sub-man; I'd recommend to keep it as-is and discuss the message here separately from this PR